src/shared/js/helpers/blur.js

```diff
@@ -1,4 +1,9 @@
-export const setBlur = ($el, editUrl) => {
+export const setBlur = ($target, editUrl) => {
+  let $el = $target;
+  if (!$el.data('cskEntryId')) {
+    const $parentEl = $target.parents('[data-csk-entry-id]');
```

Comment:
What is the purpose of this? Sometimes there are areas which do not have a specific ID, but are there for logical groupings.

Reply:
The purpose of this is to always trigger the outline for the root of the content item. 
In order to prevent the outline from jumping around the elements that are actually just a field of the parent one.
See my comment here: https://lastrev.atlassian.net/browse/SIDEKICK-16